numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -38,6 +38,40 @@
 #include "npy_cblas.h"
 #include "npy_buffer.h"
 
+
+/*
+ * Define array fields with only the flags set. The flags are set to preserve
```

Comment:
Whoops, outdated, will update as soon as anyone says this seems not completely unreasonable ;).